Plugins~/Src/MeshSyncClientBlender/python/unity_mesh_sync_baking.py

```diff
@@ -369,6 +373,13 @@ def preBakeObject(self, obj):
                 # Can't apply modifiers with shared data:
                 bpy.ops.object.make_single_user(type='SELECTED_OBJECTS', obdata=True)
                 for mod in obj.modifiers[:]:
+                    if bakeSettings.realize_instances and mod.type == "NODES":
+                        output = mod.node_group.nodes["Group Output"]
```

Comment:
That name is not guaranteed to exist, users can rename nodes. I think you'll need to loop through the nodes and find nodes with the output type and probably check which is the active output in case there are more than 1. I'm doing something similar in `findMaterialOutputNode` for the material graph.

Reply:
I changed it to use the type. I could not find a reliable way to get the right group output, since active would point to the last selected output node or -1 if none has been selected yet. So instead, I just iterate over all output nodes and add the realize instances, assuming it will be a small number.